Source/WebCore/platform/graphics/cv/VideoFrameCV.mm

```diff
@@ -344,6 +346,17 @@
     callback({ });
 }
 
+void VideoFrame::paintInContext(GraphicsContext& context, const FloatRect& destination, bool shouldDiscardAlpha)
+{
+    // FIXME: Handle alpha discarding.
+    UNUSED_PARAM(shouldDiscardAlpha);
+
+    auto conformer = makeUnique<PixelBufferConformerCV>((__bridge CFDictionaryRef)@{ (__bridge NSString *)kCVPixelBufferPixelFormatTypeKey: @(kCVPixelFormatType_32BGRA) });
```

Comment:
It is really unfortunate you have to create a conformer every time a frame is drawn to a context.

Reply:
Right, in practice, I believe the code path will be different since painting should happen in GPUProcess.
I'll add a FIXME, maybe the context should own its conformer.